routes/question.route.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const app = express();
+const questionRoute = express.Router();
+
+//Question model
+let Question = require('../models/Question');
+
+//Get All Questions
+questionRoute.route('/').get((req, res) => {
```

Comment:
I'm not familiar with this syntax for routes. Lets meet up on this and see which routing will work best for us. Take a look at my User-Schema pull request to see what I used for the routes. Might not matter which we use.